build/templates/Build_and_UnitTest.yml

```diff
@@ -413,13 +413,14 @@ steps:
     TargetFolder: "$(CIRoot)\\PLOC\\$(Build.SourceBranchName)\\$(Build.BuildNumber)"
   condition: " and(succeeded(),eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true')) "
 
-- task: PowerShell@1
-  displayName: "Publish Artifacts to MyGet"
-  continueOnError: "true"
+- task: NuGetCommand@2
```

Comment:
which NuGet version does this use?
I imagine it rolls automatically?

Reply:
At the beginning of the job, we have this:

https://github.com/NuGet/NuGet.Client/blob/2eab204630bd544e15899c67c733af3015092828/build/templates/Build_and_UnitTest.yml#L23-L26

We should see if there's a "latest version" option, but we can do that in another PR.